cam.go

```diff
@@ -254,6 +263,11 @@ func (fc *filteredCamera) images(ctx context.Context, extra map[string]interface
 
 func (fc *filteredCamera) shouldSend(ctx context.Context, img image.Image) (bool, error) {
 	for _, vs := range fc.visionServices {
+		// should not send if the vision service is an inhibitor
+		if fc.inhibitors[vs.Name().Name] {
+			return false, nil
+		}
```

Comment:
It's not quite this simple -- it requires you to check the classifications or detections, and if the detections/classifications are true, then you specifically DO NOT send. 

I believe this just simply returns false if a vision service is set to "inhibit: true"